include/arch/arm/aarch32/cortex_a_r/cpu.h

```diff
@@ -48,6 +48,7 @@
 #define DFSR_AXI_SLAVE_MASK	(1 << 12)
 
 /* Armv8-R AArch32 architecture profile */
+#define VBAR_Msk (0xFFFFFFF0U)
```

Comment:
```suggestion
#define VBAR_Msk (0xFFFFFFE0U)
```

It would also be nice to keep it consistent with the rest in this file.
```suggestion
#define VBAR_MASK		(0xFFFFFFE0U)
```

Reply:
Thanks @stephanosio looks like I misread documentation. Bits [4:0] Reserved, RES0. bit(4) included :)